### PR TITLE
clean code

### PR DESCRIPTION
**Pull request by kenkinky**
easier to use

### CODE REVIEW

File: NFServer/NFGameServerPlugin/NFCreateRoleModule.cpp [hunk 1]
```diff
@@ -278,14 +280,16 @@ void NFCreateRoleModule::AttachData(const NFGUID & self)
...
 			if (xPropManager)
 			{
-				ConvertPBToPropertyManager(it->second.property(), xPropManager);
+				pCommonRedisModule->ConvertPBToPropertyManager(it->second.property(), xPropManager);
 			}
 
 			if (xRecordManager)
 			{
-				ConvertPBToRecordManager(it->second.record(), xRecordManager);
+				pCommonRedisModule->ConvertPBToRecordManager(it->second.record(), xRecordManager);
```
Comment (ketoo) [inline NFServer/NFGameServerPlugin/NFCreateRoleModule.cpp:292]: NFGameServerPlugin/NFCreateRoleModule.cpp 

We can't use common Redis module here as normally we don't load the Redis plugin when launching the game server.

A good way is to let the function named "ConvertPBToPropertyManager" and "ConvertPBToRecordManager" as static functions or we find a safe way.

Comment (kenkinky) [reply NFServer/NFGameServerPlugin/NFCreateRoleModule.cpp]: okey, static functions coming...

File: NFComm/NFNetPlugin/NFNetPlugin.vcxproj [hunk 2]
```diff
@@ -76,14 +76,6 @@
...
   <ImportGroup Label="ExtensionSettings">
   </ImportGroup>
-  <ImportGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="PropertySheets">
-    <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
-    <Import Project="..\NFCommon.props" />
-  </ImportGroup>
-  <ImportGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="PropertySheets">
-    <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
-    <Import Project="..\NFCommon.props" />
-  </ImportGroup>
```
Comment (ketoo) [inline NFComm/NFNetPlugin/NFNetPlugin.vcxproj:86]: Could you please recover this config data as now is be used for VS projects.

Comment (kenkinky) [reply NFComm/NFNetPlugin/NFNetPlugin.vcxproj:86]: Why I can't do that? those code repeated with line 53~59. because I get a compile warning.

Comment (ketoo) [reply NFComm/NFNetPlugin/NFNetPlugin.vcxproj:86]: Great. We would do the removal if we have repeated config data.

**Review by ketoo — APPROVED** [review]

**Pull request merged**